packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -5,6 +5,8 @@ import {
   initializeWithHeadless,
 } from 'c/quanticHeadlessLoader';
 
+import search from '@salesforce/label/c.quantic_Search';
```

Comment:
I think I would create the `labels = { search }` even if there is just one, might be a bit confusing seeing only `${search}...` in the code.